Makefile

```diff
@@ -0,0 +1,104 @@
+# Makefile of the Arch Linux Updates Indicator GNOME Shell Extension
+# (adapted from https://gitlab.com/skrewball/openweather/)
+
+PKG_NAME = arch-update
+UUID = arch-update@RaphaelRochet
+BASE_MODULES = metadata.json LICENCE.txt prefs.xml
+SRC_MODULES = extension.js prefs.js stylesheet.css
+EXTRA_DIRECTORIES = icons
+TOLOCALIZE = extension.js prefs.js
+MSGSRC = $(wildcard locale/*/arch-update.po)
+
+# Packagers: Use DESTDIR for system wide installation
+ifeq ($(strip $(DESTDIR)),)
+	INSTALLTYPE = local
+	INSTALLBASE = $(HOME)/.local/share/gnome-shell/extensions
+else
+	INSTALLTYPE = system
+	SHARE_PREFIX = $(DESTDIR)/usr/share
+	INSTALLBASE = $(SHARE_PREFIX)/gnome-shell/extensions
+endif
+# Set a git version for self builds from the latest git tag with the revision
+# (a monotonically increasing number that uniquely identifies the source tree)
+# and the current short commit SHA1. (Note: not set if VERSION passed)
+GIT_VER = $(shell git describe --long --tags | sed 's/^v//;s/\([^-]*-g\)/r\1/;s/-/./g')
+# The command line passed variable VERSION is used to set the version integer
+# in the metadata and in the generated zip file. If no VERSION is passed, we
+# won't touch the metadata version and instead use that for the zip file.
+ifdef VERSION
+	ZIPVER = -v$(VERSION)
+else
+	ZIPVER = -v$(shell cat metadata.json | sed '/"version"/!d' | sed s/\"version\"://g | sed s/\ //g)
+endif
+
+.PHONY: all clean extension potfile mergepo install install-local zip-file
+
+all: extension
+
+clean:
+	rm -f ./schemas/gschemas.compiled
+	rm -f ./locale/*/arch-update.po~
+	rm -f ./locale/*/LC_MESSAGES/arch-update.mo
+
+extension: ./schemas/gschemas.compiled $(MSGSRC:arch-update.po=LC_MESSAGES/arch-update.mo)
+
+./schemas/gschemas.compiled: ./schemas/org.gnome.shell.extensions.arch-update.gschema.xml
+	glib-compile-schemas ./schemas/
+
+# TODO: disabled, since xgettext is not able to include prefs.xml to the pot file
```

Comment:
I use this CLI : 
xgettext --from-code=UTF-8 -j -o "$POTFILE" -L glade *.xml

Reply:
Thank you! I couldn't reproduce the binaries. What's your `msgfmt` command line?

Reply:
I use `msgfmt "$pofile" -o "$mofile"`, no checking. In rare cases of something wrong, I let translators come back and fix it.